src/nvim/editor/arbmark.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Same function but names are not limited to one char
+ * There is also no ex_command, just viml functions
+ */
+
+#include "nvim/vim.h"
+#include "nvim/globals.h"
+#include "nvim/mark.h"
+#include "nvim/memory.h"
+#include "nvim/map.h"
+#include "nvim/lib/kvec.h"
+#include "nvim/editor/arbmark.h"
```

Comment:
It might be worth pointing out here that API functions will soon be accessible from VimL, though with different naming and type-checking conventions than native VimL builtins. The benefit is that the generated wrappers will make (most of) the type-checking and conversion for you.